### ISSUE TITLE
KeyboardLayoutObject type not allowing custom layouts

### ISSUE THREAD

**Issue by RedStarThrower** [hodgef/simple-keyboard#61]
**Simple-keyboard version**
2.20.3

**Describe the bug**
In your [demo example](https://codesandbox.io/s/github/hodgef/simple-keyboard-js-demos/tree/mobile/), you add a custom layout name to the layout object. However, the interface does not support it, and is impossible to extend, as it is not exported.

```TypeScript
  interface KeyboardLayoutObject {
    default: string[];
    shift?: string[];
  }
```
Perhaps it should be like this?

```TypeScript
 interface KeyboardLayoutObject {
    default: string[];
    shift?: string[];
    [key: string]: string[];
  }
```

**Comment by hodgef** [comment]
Hello @RedStarThrower,

Thanks for reporting this issue! This will be fixed shortly.

Regards,
Francisco Hodge

**Comment by hodgef** [comment]
Hello @RedStarThrower,

A fixed version (simple-keyboard `2.20.4`) has been released. Feel free to try it out!

Thank you,
Francisco Hodge

**Comment by RedStarThrower** [comment]
Worked like a charm; thank you for the awesomely quick fix and release!
